tasks/post-to-cmr/index.js

```diff
@@ -122,14 +156,11 @@ async function postToCMR(event) {
 
   const startTime = Date.now();
 
-  const cmrSettings = await getCmrSettings({
-    ...event.config.cmr,
-    ...event.config.launchpad,
-  });
-
   // post all meta files to CMR
-  const results = await Promise.all(
-    updatedCMRFiles.map((cmrFile) => publish2CMR(cmrFile, cmrSettings, cmrRevisionId))
+  const results = await pMap(
+    updatedCMRFiles,
+    (cmrFile) => publish2CMR(cmrFile, cmrSettings, cmrRevisionId),
+    { concurrency }
```

Comment:
`stopOnError: true` is default, if set to `false`, it will change current lambda behavior